content/hi/docs/reference/glossary/kube-proxy.md

```diff
@@ -0,0 +1,27 @@
+---
+title: kube-proxy
+id: kube-proxy
+date: 2018-04-12
+full_link: /docs/reference/command-line-tools-reference/kube-proxy/
+short_description: >
+  `kube-proxy` एक नेटवर्क प्रॉक्सी है जो क्लस्टर में प्रत्येक नोड पर चलता है।
+
+aka:
+tags:
+- fundamental
+- networking
+---
+kube-proxy एक नेटवर्क प्रॉक्सी है जो आपके क्लस्टर में प्रत्येक 
+{{< glossary_tooltip text="नोड" term_id="node" >}} पर चलता है,
```

Comment:
This line is more appropriate:-

Line 15: {{< glossary_tooltip text="नोड" term_id="node" >}} पर चलती है,